.github/workflows/doc-tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: 6.0.x
+        dotnet-version: 8.0.x
```

Comment:
Okay now we really are installing the same version of .NET twice in several workflows, and my comments from an earlier PR to remove the duplication actually apply now :)